cclib/parser/turbomoleparser.py

```diff
@@ -149,9 +149,82 @@ def extract(self, inputfile, line):
                     self.set_attribute('atomcoords', atomcoords)
                     self.set_attribute('atomnos', atomnos)
 
+        # Frequency values in aoforce.out
+        #        mode               7        8        9       10       11       12
+        #
+        #      frequency          53.33    88.32   146.85   171.70   251.75   289.44
+        #
+        #      symmetry            a        a        a        a        a        a
+        #
+        #         IR               YES      YES      YES      YES      YES      YES
+        # |dDIP/dQ|   (a.u.)     0.0002   0.0000   0.0005   0.0004   0.0000   0.0000
+        # intensity (km/mol)       0.05     0.00     0.39     0.28     0.00     0.00
+        # intensity (  %   )       0.05     0.00     0.40     0.28     0.00     0.00
+        #
+        #        RAMAN             YES      YES      YES      YES      YES      YES
+        #
+        #   1   c           x   0.00000  0.00001  0.00000 -0.01968 -0.04257  0.00001
+        #                   y  -0.08246 -0.08792  0.02675 -0.00010  0.00000  0.17930
+        #                   z   0.00001  0.00003  0.00004 -0.10350  0.11992 -0.00003
+        if 'NORMAL MODES and VIBRATIONAL FREQUENCIES (cm**(-1))' in line:
+            vibfreqs = []
+            vibsyms = []
+            vibirs = []
+            vibdisps = []
+            while '****  force : all done  ****' not in line:

```

Comment:
Using `while` in this capacity can be problematic, although I'm not strongly opposed. An alternative and maybe more dependable stopping condition might be to count the number of frequencies/displacements parsed. We know how many there should be, right? You could also exit early if this line is there for some reason and not all were printed.

Reply:
Well, we can stop once we get the number of frequencies. But the thing is that Turbomole prints frequencies with value 0.0 as well, which get parsed and need to be deleted later and all the corresponding `vibdisps` and `vibirs` need to be deleted too. So I am not sure if counting till the number is the best way to go.

Reply:
I understand these are at the end only to fill up a line?

Reply:
We discussed this offline, and opted for keeping the while in place here.